benchmark/README.md

```diff
@@ -1,32 +1,34 @@
-### Benchmark for HGT detection software
-
-## Scripts details
-
-	* `launch_software.sh` launch all tested software
-	* `reformat_input.py` reformats input data (ex. species
-		and gene trees in Newick format) to input supported
-		by each software (ex. Nexus format)
-	* `parse_output.py` parses the output of each software
-		to report the number of HGTs detected
-	* `compute_accuracy.py` uses the output of `launch_software.sh`
-		to compute the number of true positive, false positive
-		and false negative HGTs, as well as precision, recall
-		and F-score for each software
-	* `test_1_simulate_genomes.sh` simulates various data sets
-		using the [ALF simulator](http://mbe.oxfordjournals.org/content/early/2011/12/07/molbev.msr268)
+## Benchmark for HGT detection software
```

Comment:
This is not parsing correctly:
https://github.com/ekopylova/WGS-HGT/blob/issue_19/benchmark/README.md